scene/register_scene_types.cpp

```diff
@@ -712,6 +712,9 @@ void register_scene_types() {
 	String font_path = GLOBAL_DEF("gui/theme/custom_font", "");
 	ProjectSettings::get_singleton()->set_custom_property_info("gui/theme/custom_font", PropertyInfo(Variant::STRING, "gui/theme/custom_font", PROPERTY_HINT_FILE, "*.tres,*.res,*.font", PROPERTY_USAGE_DEFAULT | PROPERTY_USAGE_RESTART_IF_CHANGED));
 
+	ResourceLoader::set_timestamp_on_load(true);
+	ResourceSaver::set_timestamp_on_save(true);
```

Comment:
That will cause it to be true when running games too, and not just in the editor, which I fear is not a good idea.

Reply:
Well, I was suspecting something like that, but the use of the boolean set by these functions are in  `TOOLS_ENABLED` ifdef blocks. Thus the timestamp is only set/changed if this define is set and I was guessing it was only for editor mode.

Reply:
It's probably cleaner to put those two lines in `TOOLS_ENABLED` block to avoid setting the boolean, even if it won't be used eventually.

Reply:
Actually now that I think about it, adding `TOOLS_ENABLED` is not enough. An editor binary has `TOOLS_ENABLED` defined, but it's not necessarily the editor - you can use it to run games (and it's used to run games from within the editor). In this case we don't want it to mess with timestamps either.

So there should also be a `Engine::get_singleton()->is_editor_hint()` check (which is also true for the project manager).